### PR TITLE
[benchmark] Fixes #125  Add PoseNet into Benchmark Test.

### LINKED ISSUE #125: [test] Support Benchmark Test with PoseNet.

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin PTAL, thanks.

### CODE REVIEW

File: examples/posenet/src/PoseNet.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class PoseNet{
-  constructor(modelArch, backend, version, outputStride, inputShape, type, cacheMap) {
+  constructor(modelArch, version, outputStride, inputShape, type, cacheMap, backend) {
```
Comment (huningxin) [inline examples/posenet/src/PoseNet.js]: What's the reason to change the signature? Did we change every reference of this API?

Comment (BruceDai) [reply examples/posenet/src/PoseNet.js]: Changing this signature for invoking PoseNet adapt for Benchmark test, let backend as default function parameter, if backend had no value, undefined is passed.

Comment (huningxin) [reply examples/posenet/src/PoseNet.js]: sounds good.

**Comment by huningxin** [comment]
Looks good. Thanks!

**Pull request merged**